### DISCUSSION TITLE
Video filters on Android

### DISCUSSION THREAD

**Discussion by HappyDev96** [TheWidlarzGroup/react-native-video#4940] [Ideas]
### What problem are you trying to solve?

I want to add video filters and I'm working on Android. When I read the doc of this package, I could see that there are filters for only iOS. I want to know if it's possible to add video filters on Android.

### Proposed solution

Add support for video filters on Android (equivalent to the iOS-only filter capability currently documented).

### Platforms

Android

**Comment by awesomeo99** [comment]
do you have any code on how you are trying to add filters? which library are you using?

**Comment by HappyDev96** [comment]
You can see filterEnabled and filters props in this package document.

**Comment by HappyDev96** [comment]
Those props are only available for ios.
But I want to use those props for Android.
Is there anyway for it?

**Comment by karanpratapsingh** [comment]
@HappyDev96 did you find any solution?

**Comment by ajaxsys** [comment]
@HappyDev96 @karanpratapsingh same needs any solutions？

**Comment by CHaNGeTe** [comment]
Try this PR
https://github.com/react-native-community/react-native-video/pull/1671

**Comment by karanpratapsingh** [comment]
@ajaxsys No, unfortunately , but try the #1671 PR

**Comment by ShashwatMDas** [comment]
Any updates on this? I'm trying to do the same thing, add filters over videos for both android and ios app just like Instagram does. If anyone was able to do this, can you help me out?

**Comment by mouzakim** [comment]
Any updates excluding the PR #1671 ?

**Comment by isthisshivam** [comment]
anyone solve this fuck please let me know how to use fliters in android 
Thanks in advance !

**Comment by Aijaz12550** [comment]
I am also facing this issue. any solution?

**Comment by isthisshivam** [comment]
No I haven't heard anything about this 😭

**Comment by izuutech** [comment]
Any solution for adding filters in android yet?

**Comment by freeboub** [comment]
Linked pr is a good start, but someone need to port it on master

**Comment by Space6188** [comment]
Any news?

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
